src/scvmm/HISTORY.rst

```diff
@@ -26,3 +26,8 @@ Release History
 * Force and retain flags in VM delete
 * Generated SDK now requires updated version of azure-cli
 * Long running PATCH operations
+
+0.1.5
++++
+* Requesting VMMServer credentials from the user until they are non-empty.
+* Removing default value for port. Asking for the input. If input is empty, setting port to 8100.
```

Comment:
Removing the default value will cause breaking change, right? If so, please add `[BREAKING CHANGE]` as the prefix
```suggestion
* [BREAKING CHANGE] Removing default value for port. Asking for the input. If input is empty, setting port to 8100.
```